package.json

```diff
@@ -86,6 +86,7 @@
     "glob": "^7.1.1",
     "gts": "^0.9.0",
     "handlebars": "^4.0.11",
+    "htm": "^2.1.1",
```

Comment:
what are these for?

Reply:
These are used for templating fixtures in unit testing.